test/integration/external-accounts.test.js

```diff
@@ -448,12 +450,21 @@ describe("External Accounts", function () {
 
   describe("PATCH /external-accounts/link/:token", function () {
```

Comment:
Integration test doesn't check if the role actually got removed, add that in the success case

Reply:
I've added it. Can you please check it?